Library/Homebrew/dev-cmd/generate-cask-api.rb

```diff
@@ -0,0 +1,62 @@
+# typed: true
+# frozen_string_literal: true
+
+require "cli/parser"
+require "cask/cask"
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  sig { returns(CLI::Parser) }
+  def generate_cask_api_args
+    Homebrew::CLI::Parser.new do
+      description <<~EOS
+        Generates Cask API data files for formulae.brew.sh.
+      EOS
+
+      named_args :none
+    end
+  end
+
+  JSON_TEMPLATE = <<~EOS
+    ---
+    layout: cask_json
+    ---
+    {{ content }}
+  EOS
```

Comment:
I think eventually it would be nice to just bypass Jekyll for the JSON stuff tbh. Only then can the JWS signing stuff move here.